web/Controllers/LandlordController.cs

```diff
@@ -221,6 +256,7 @@ public async Task<IActionResult> ViewProperties(int? id = null, int page = 1, in
             new PropertyListModel(listOfProperties, properties.Count, landlordProfile, page));
     }
 
+    [Authorize(Roles = "Admin,Landlord")]

```

Comment:
same again, and a few below (not going to comment on all of them)

Reply:
Ok, I'll try to find all of them.